syntax = "proto3";

package fbsrankings.messages.query;

message AffiliationCountBySeasonResult {
    string query_id = 1;
    string season_id = 2;
    int32 fbs_count = 3;
    int32 fcs_count = 4;
}

message AffiliationCountBySeasonQuery {
    string query_id = 1;
    string season_id = 2;
}
